README.md

```diff
@@ -43,7 +43,20 @@ Als externe Libraries werden benötigt:
  * Adafruit Unified Sensor, Version 1.02
 
 Für LoRa zusätzlich:
- * MCCI LoRaWAN LMIC library, Version 2.3.2
+ * MCCI LoRaWAN LMIC library, Version >= 2.3.2  
+**Achtung:** Bitte prüfen, dass in der Datei  project_config/lmic_project_config.h (in der obersten Ebene in dieser Library) unbedingt
```

Comment:
ist das wirklich so vorgesehen, dass man da im library source rumfrickeln muss?

oder koennte man auch den include-suchpfad so aendern, dass eine fuer uns passende header-datei (die in unserem git-repo committed ist) zuerst gefunden wird und anstelle o.g. verwendet wird?

Reply:
- Keine Ahnung wie man in der (blöden) Arduino-IDE den Include-Pfad anpassen kann
- wenn die Library-Maintainer die #define mal ändern, sitzen wir auf mit einer eigenen include (zugegeben ist das Risiko sehr klein)

Reply:
das ist richtig, tritt aber hoffentlich nicht so oft auf.

andererseits muesste man o.g. frickelei ja jedes mal machen, wenn man die library updated, weil dann die aenderung wieder ueberschrieben werden.

Reply:
Und was machen wir jetzt?

Reply:
Hab das im lmic source gefunden:
```
// We need to be able to compile with different options without editing source.
// When building with a more advanced environment, set the following variable:
// ARDUINO_LMIC_PROJECT_CONFIG_H=my_project_config.h
//
// otherwise the lmic_project_config.h from the ../../project_config directory will be used.
#ifndef ARDUINO_LMIC_PROJECT_CONFIG_H
# define ARDUINO_LMIC_PROJECT_CONFIG_H ../../project_config/lmic_project_config.h
#endif

```

Reply:
d.h. das headerfile in unser git repo aufnehmen und die environment variable setzen, das ist wohl der vorgesehene weg.

Reply:
das steht im lib readme:

## Configuration

A number of features can be enabled or disabled at compile time.
This is done by adding the desired settings to the file
`project_config/lmic_project_config.h`. The `project_config`
directory is the only directory that contains files that you
should edit to match your project; we organize things this way
so that your local changes are more clearly separated from
the distribution files. The Arduino environment doesn't give
us a better way to do this, unless you change `BOARDS.txt`.

Unlike other ports of the LMIC code, in this port, you should not edit `src/lmic/config.h` to configure this package.

The following configuration variables are available.

### Selecting the LoRaWAN Region Configuration

The library supports the following regions:

`-D` variable | CFG region name | CFG region value | LoRaWAN Regional Spec 1.0.3 Reference| Frequency
------------|-----------------|:----------------:|:-------------------:|--------
`-D CFG_eu868` | `LMIC_REGION_eu868` | 1 | 2.2 | EU 863-870 MHz ISM
`-D CFG_us915` | `LMIC_REGION_us915` | 2 | 2.3 | US 902-928 MHz ISM
`-D CFG_au921` | `LMIC_REGION_au921` | 5 | 2.6 | Australia 915-928 MHz ISM
`-D CFG_as923` | `LMIC_REGION_as923` | 7 | 2.8 | Asia 923 MHz ISM
`-D CFG_as923jp` | `LMIC_REGION_as923` and `LMIC_COUNTRY_CODE_JP` | 7 | 2.8 | Asia 923 MHz ISM with Japan listen-before-talk (LBT) rules
`-D CFG_kr920` | `LMIC_REGION_kr920` | 8 | 2.9 | Korea 920-923 MHz ISM
`-D CFG_in866` | `LMIC_REGION_in866` | 9 | 2.10 | India 865-867 MHz ISM

You should define exactly one of `CFG_...` variables. If you don't,
the library assumes `CFG_eu868`. The library changes configuration pretty substantially
according to the region. Some of the differences are listed below.